src/System.Security.Permissions/src/System/Security/Permissions/PrincipalPermission.cs

```diff
@@ -78,7 +78,7 @@ public bool IsSubsetOf(IPermission target)
             }
             else if (!VerifyType(target))
             {
-                throw new ArgumentException(SR.Argument_WrongType, GetType().FullName);
+                throw new ArgumentException(SR.Format(SR.Argument_WrongType, GetType().FullName), GetType().FullName);
```

Comment:
These probably should be

```c#
throw new ArgumentException(SR.Format(SR.Argument_WrongType, GetType().FullName), nameof(target));
```

The versions in .NET Framework all left out the paramName; but I see no harm in it saying "target".

Reply:
for all ArgumentException type?